RFC/RFC-14-p2p-adapter.md

```diff
@@ -90,70 +91,79 @@ sequenceDiagram
     participant Peer5
     participant RS as Routing Service
 
-    RS->>Peer1: { peer2: [], peer3: [], peer4: [] }
-    RS->>Peer2: { peer1: [], peer3: [], peer4: [ peer1 ] }
-    RS->>Peer3: { peer1: [], peer2: [], peer4: [ peer1 ] }
-    RS->>Peer4: { peer1: [], peer2: [ peer1 ], peer3: [ peer1 ] }
-    RS->>Peer5: {}
+    RS->>Peer1: { paths: [ [1, 2], [1, 3], [1, 4] ], unreachable: [5] }
```

Comment:
I don't think the routing service should be explicit about the unreachable routes, since the peer knows every peer, then it can deduce from the peers there is no route to, they are unrecheable